addons/web/static/src/views/list/list_confirmation_dialog.js

```diff
@@ -92,4 +102,21 @@ export class ListConfirmationDialog extends Component {
     isValueOperation(field) {
         return this.props.changes[field.name] instanceof Operation;
     }
+
+    formatDate(date) {
+        const dd = String(date.getDate()).padStart(2, "0");
+        const mm = String(date.getMonth() + 1).padStart(2, "0");
+        const yyyy = date.getFullYear();
+        return `${dd}/${mm}/${yyyy}`;
+    }
+
+    get today() {
+        return this.formatDate(new Date());
```

Comment:
use `formatDate` from core/l10n/dates:
```suggestion
        return formatDate(luxon.DateTime.now())
```

Reply:
why change in the manifest? luxon is already there